client/components/Editor/plugins/discussions/discussionsState.ts

```diff
@@ -172,6 +159,19 @@ export const createDiscussionsState = (options: Options) => {
 		});
 	};
 
+	const addDiscussion = (discussionId: string, selection: DiscussionSelection) => {
```

Comment:
This needs to call `asynchronouslyUpdateDiscussion`, not `updateDiscussion`, hence the move